pkgs/development/haskell-modules/hackage-packages.nix

```diff
@@ -118602,6 +118602,24 @@ self: {
        hydraPlatforms = lib.platforms.none;
      }) {};
 
+  "happy_1_19_12" = callPackage
+    ({ mkDerivation, array, base, Cabal, containers, directory
+     , filepath, mtl, process
+     }:
+     mkDerivation {
+       pname = "happy";
+       version = "1.19.12";
+       sha256 = "03xlmq6qmdx4zvzw8bp33kd9g7yvcq5cz4wg50xilw812kj276pv";
+       isLibrary = false;
+       isExecutable = true;
+       setupHaskellDepends = [ base Cabal directory filepath ];
+       executableHaskellDepends = [ array base containers mtl ];
+       testHaskellDepends = [ base process ];
+       description = "Happy is a parser generator for Haskell";
+       license = lib.licenses.bsd2;
+       hydraPlatforms = lib.platforms.none;
+     }) {};
+
```

Comment:
Maybe I just missed it, but did you add this by hand?

If so, you'll have to instead add it to `extra-packages` in `pkgs/development/haskell-modules/configuration-hackage2nix/main.yaml` and regenerate the Haskell package set.

Reply:
Ok, I will fix this.

Reply:
fixed..